src/components/Navbar/Navbar.cy.tsx

```diff
@@ -0,0 +1,89 @@
+import Navbar, { NavbarItem } from "./Navbar";
+import {
+  faClock,
+  faCompass,
+  faFutbol,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
+
+const initialItems = [
+  {
+    icon: faClock,
+    title: "Time",
+  },
+  {
+    icon: faFutbol,
+    title: "Comments",
+  },
+  {
+    icon: faCompass,
+    title: "Compass",
+  },
+  {
+    icon: faUser,
+    title: "User",
+  },
+];
+
+describe("<Navbar />", () => {
+  it("renders navbar with 4 items", () => {
```

Comment:
todo: This test you say that it renders with 4 items, but it's missing an assertion to actually test 4 items in the DOM ;)

Reply:
hehe, updated here